ocp_resources/resource.py

```
@@ -76,6 +84,82 @@ def _get_api_version(dyn_client: DynamicClient, api_group: str, kind: str) -> st
     return res.group_version
 
 
+def client_configuration_with_basic_auth(
```

Pylint (3.3.7)

[refactor] Too many local variables (23/15)

(R0914)